contracts/common/misc/Drainable.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.5.2;
+
+import { Governable } from "../governance/Governable.sol";
+
+import {IERC20} from "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
+
+
+contract Drainable is Governable {
+
+  constructor (address _governance) public Governable(_governance) {}
+
+  function drainErc20 (
+    address[] calldata tokens,
```

Comment:
Let's also accept the `uint256[] amount`. Better to be flexible. Same for the next 2 functions.